src/pages/projects/edit/Edit.tsx

```diff
@@ -65,6 +65,20 @@ export function Edit() {
         />
       </Element>
 
+      <Element leftSide={t('client')}>
+        <DebouncedCombobox
+          defaultValue={project?.client_id}
+          endpoint="/api/v1/clients"
+          label="display_name"
+          onChange={(value) => handleChange('client_id', value.value)}
+          clearButton={Boolean(project?.client_id)}
+          onClearButtonClick={() => handleChange('client_id', '')}
+          errorMessage={errors?.errors.client_id}
+          queryAdditional
+          disabled
+        />
+      </Element>
```

Comment:
We have `ClientSelector` component in the app. Please make use of it.

Reply:
It's replaced!